content/case_histories/hoop_region_norway/processinginterpretation.rst

```diff
@@ -0,0 +1,210 @@
+
+The simplest approach to achieving this is to upscale both seismic and CSEM results. The reservoir parameter that is best constrained by the CSEM method is the transverse resistance (vertically integrated resistivity). By calculating the transverse resistance from the seismically and CSEM derive resistivity, the difference in vertical resolution between the two methods can be overcome, albeit at the expense of the higher resolution of the seismic result. Transverse resistances based on CSEM and seismic results are compared in :numref:`fig_transverse_resistivity_compare` a. Note that when the resistivity is calculated from the seismic data using the Simandoux relationship, which is calibrated to horizontal resistivity at the well, the result is most closely related to the horizontal resistivity of the sub-surface. The CSEM measurements, in contrast, provide a measure of both horizontal and vertical resistivity however reservoir related structure manifests in the vertical resistivity. To address this difference, an empirical calibration factor of three, based on CSEM analysis of background anisotropy, was applied to the seismically derived transverse resistance to compensate for the electrical anisotropy observed.
+
+An analysis of :numref:`fig_transverse_resistivity_compare` a offers important information about the hydrocarbon saturation levels of the reservoirs and potential reservoirs along the section. A good agreement is seen between the CSEM and seismic transverse resistance curves at the end member positions represented by the Central well location (Sw=0.1 - magenta curve) and for the Alternative well location (Sw=1 - blue curve) that corroborate the validity of the rock physics model used and the calibration factor applied to the data. For the case of the Bjaaland well, the CSEM derived transverse resistance most closely agrees with the lower saturation seismically derived curves, indicating, in a semi-quantitative way, the absence of a commercial hydrocarbon saturation at this location. Finally, for the Hanssen well, the separation between the CSEM transverse resistance curve and that for the wet case seismically-derived curve (blue curve) indicates the presence of hydrocarbon saturation at least higher than 50%. However, it is important to have in mind that a diminished CSEM-derived resistivity value is expected to be found in this area due to the 3D effects discussed previously.   As a consequence of this the hydrocarbon saturation will be underestimated in this case.
+
+.. figure:: images/transverse_resistivity_compare.png
+    :align: center
+    :figwidth: 100%
+    :name: fig_transverse_resistivity_compare
+
+a) Comparison between the seismically- and CSEM-derived transverse resistance estimates.
```

Comment:
If you first add a statement, eg:
```
Transverse resistivity comparison. (a) .... (b) .... (c) .... (d)....  
```
it will work. This way the figure also gets numbered